platform/mikro-e/Makefile.mikro-e

```diff
@@ -50,3 +54,21 @@ CFLAGS += -DRF_CHANNEL=$(CHANNEL)
 CFLAGS += -DIEEE802154_CONF_PANID=$(PAN_ID)
 
 include $(CONTIKI)/cpu/pic32/Makefile.pic32
+
+CUSTOM_RULE_LINK=1
+
+### CPU-dependent cleanup files
+CLEAN += *.hex *.$(TARGET)
+
+.PHONY: bin2hex
+
+all: $(CONTIKI_PROJECT).$(TARGET) bin2hex
```

Comment:
When using all you can do it much easier, you can get rid of the CUSTOM_RULE_LINK and the custom rule below and just do it like this (see full patch below):

all: $(CONTIKI_PROJECT).$(TARGET)
    xc32-bin2hex $(CONTIKI_PROJECT).mikro-e

This is the kind of a fix I was thinking of, it's just in this case we have to get rid of the dependency on CONTIKI_PROJECT. 

```diff --git a/platform/mikro-e/Makefile.mikro-e b/platform/mikro-e/Makefile.mikro-e
index d2bea39..6fdb602 100755
--- a/platform/mikro-e/Makefile.mikro-e
+++ b/platform/mikro-e/Makefile.mikro-e
@@ -55,20 +55,15 @@ CFLAGS += -DIEEE802154_CONF_PANID=$(PAN_ID)
 
 include $(CONTIKI)/cpu/pic32/Makefile.pic32
 
-CUSTOM_RULE_LINK=1
+#CUSTOM_RULE_LINK=1
 
 ### CPU-dependent cleanup files
 CLEAN += *.hex *.$(TARGET)
 
-.PHONY: bin2hex
-
-all: $(CONTIKI_PROJECT).$(TARGET) bin2hex
-
-$(CONTIKI_PROJECT).$(TARGET): $(CONTIKI_PROJECT).co $(PROJECT_OBJECTFILES) $(PROJECT_LIBRARIES) contiki-$(TARGET).a
-       $(TRACE_LD)
-       $(Q)$(LD) $(LDFLAGS) $(TARGET_STARTFILES) ${filter-out %.a,$^} ${filter %.a,$^} $(TARGET_LIBFILES) -o $@
-
-bin2hex:
-       xc32-bin2hex $(CONTIKI_PROJECT).$(TARGET)
-
+#%.$(TARGET): %.co $(PROJECT_OBJECTFILES) $(PROJECT_LIBRARIES) contiki-$(TARGET).a
+#      $(TRACE_LD)
+#      $(Q)$(LD) $(LDFLAGS) $(TARGET_STARTFILES) ${filter-out %.a,$^} \
+#              ${filter %.a,$^} $(TARGET_LIBFILES) -o $@
 
+all: $(CONTIKI_PROJECT).$(TARGET)
+       xc32-bin2hex $(CONTIKI_PROJECT).mikro-e
```